microverseinc/ror-social-scaffold
# #Final milestone-Chore/deployment

**What does this PR do?**
- this is for TSE review of implementation of the assignment.

**Task covered**
- Deploy our app to heroku.

**How can this be manually tested?**
- Visit the link  [Heroku Link](https://salty-lowlands-16667.herokuapp.com/users/sign_inurl) and you should be greeted with the app's login page.
You can also play around by creating an account and testing the other milestones covered.
